src/sc_data/data.py

```
@@ -302,12 +375,28 @@ def run(self):
+                        max_retries + 1,
+                    )
+                    self.updated.set()
+                else:
+                    # No database available, fail
+                    logger.error(
+                        "Failed to download database after %d attempts and no existing database found",
+                        max_retries + 1,
+                    )
+                    raise RuntimeError(
+                        f"Failed to download database after {max_retries + 1} attempts and no existing database available"
+                    )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)